plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -147,6 +169,26 @@ private void connect()
         this.client = client;
     }
 
+    private String prependCatalogToDbName(String dbName)
+    {
+        if (catalogName == null) {
```

Comment:
wondering, what if `catalogName=` is blank?

Reply:
Considering adding a condition to check if the catalogName is blank could be beneficial. However, ultimately, since the catalogName originates from the configuration, we can ensure proper validation there.